doajtest/helpers.py

```diff
@@ -120,6 +120,7 @@ class DoajTestCase(TestCase):
 
     @classmethod
     def setUpClass(cls) -> None:
+        import portality.app  # noqa, needed to registing routes
```

Comment:
why do we need this here if we're already doing `from portality.core import app` in the imports?

Reply:
In my env, `test_04_reject_resubmit` and other test cases will fail if without `import portality.app`.
The exception is missing routing.
```
E           werkzeug.routing.BuildError: Could not build url for endpoint 'doaj.toc' with values ['identifier']. Did you mean 'static' instead?
```


`from portality.core import app` and `import portality.app` is different.
I needed `import portality.app` `app.register_blueprint` to register routing.


**References:**
following code can list all route that registerd in app 
`cls.app_test.url_map`